uncertainties/unumpy/core.py

```diff
@@ -20,7 +20,9 @@
 
 # 3rd-party modules:
 import numpy
-from numpy.core import numeric
+from numpy import isscalar as numpy_isscalar
+from numpy import dot as numpy_dot
```

Comment:
there is no need to rename functions in imports like this. Just use "numpy.dot" (same applies to the one above)